src/Public/js/nipsweb.import.js

```diff
@@ -0,0 +1,265 @@
+/* global myradio, mConfig */
+// Queue up sending ajax requests one at a time
+var ajaxQueue = $({});
+
+//Get the GET variables for clientid & nextWeightChanneln .
+$.urlParam = function(name){
+  var results = new RegExp("[\?&]" + name + "=([^&#]*)").exec(window.location.href);
+  return results[1] || 0;
+};
+var nextWeightChannel0;
+var nextWeightChannel1;
+var nextWeightChannel2;
+
+$(document).ready(
+  function () {
+
+    nextWeightChannel0 = parseInt($.urlParam("channel0lastweight"))+1;
```

Comment:
parseInt should always be used with a radix parameter - the spec does not actually define a default number base!